drivers/hmis/app/models/hmis/ce/match/expression/client_field_map.rb

```diff
@@ -68,7 +68,7 @@ def days_since_last_exit_field
     def veteran_status_field
       {
         query: ->(clients) { clients.pluck(:id, :veteran_status).to_h },
-        format_for_display: ->(v) { HudUtility2026.veteran_status(v) },
+        format_for_display: ->(v) { HudHelper.util('2026').veteran_status(v) },
```

Comment:
```suggestion
        format_for_display: ->(v) { HudHelper.util.veteran_status(v) },
```

No need to call out 2026 here